ParisBNF/abb/BNFabb184.xml

```diff
@@ -0,0 +1,226 @@
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BNFabb11" xml:lang="en" type="mss">
+   <teiHeader xml:base="https://betamasaheft.eu/">
+      <fileDesc>
+         <titleStmt>
+            <title>Accounts about the Passions of Our Lord taken from the Gospels</title>
+            <editor role="cataloguer" key="DN"/>
+            <editor role="generalEditor" key="AB"/>
+         </titleStmt>
+         <publicationStmt>
+            <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+            <pubPlace>Hamburg</pubPlace>
+            <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+               Forschungsumgebung / Beta maṣāḥǝft</publisher>
+            <availability>
+               <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                   This file is licensed under the Creative Commons Attribution-ShareAlike 4.0.
+               </licence>
+            </availability>
+
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc xml:id="ms">
+               <msIdentifier>
+                  <repository ref="INS0303BNF"/>
+                  <collection>Manuscrits orientaux</collection>
+                  <collection>Fonds éthiopien</collection>
+                  <collection>d'Abbadie</collection>
+                  <idno>BnF Éthiopien d'Abbadie 184</idno>
+               </msIdentifier>
+               <msContents>
+                  <summary/>
+
+                  <msItem xml:id="ms_i1">
+                     <locus from="1r"/>
+                     <title type="complete">Accounts about the Passions of Our Lord from the Gospels and other texts</title>
+                     <textLang mainLang="gez"/>
+                     <note> The content of the manuscript resembles <ref type="work" corresp="LIT1544Gebrah">Lectionary of the Holy Week</ref> 
+                        (possibly incomplete) but it cannot be fullly ascertained and identified on the basis of 
+                     the available catalogue descriptions.</note>
+                     
+                     <msItem xml:id="ms_i1.1">
+                        <locus from="3r"/>
+                        <title type="incomplete" ref="LIT1558Matthew">Passions of Chirst according to Matthew (Mt 26, 47 - 27, 57)</title>
+                        </msItem>
+                     
+                     <msItem xml:id="ms_i1.2">
+                        <locus from="10v"/>
+                        <title type="incomplete" ref="LIT1882MarkGo">Passions of Chirst according to Mark (Mk 14, 17 - 20, 42)</title>
+                     </msItem>
+                     
+                     <msItem xml:id="ms_i1.3">
+                        <locus from="18v"/>
+                        <title type="incomplete" ref="LIT1812GospelLuke">Passions of Chirst according to Luke (Luke 22,47 - 23,50)</title>
+                     </msItem>
```

Comment:
I changed. I wonder how is the correct way of quoting the Scriptures and blocks thereof. I wonder if here the code should be used, like `<ref cRef="betmas:LIT2000Mazmur.11.4">Psalm 11:4</ref>`, but if it works inside the code? Maybe Zhenia will know this.

Reply:
> I changed. I wonder how is the correct way of quoting the Scriptures and blocks thereof. I wonder if here the code should be used, like `<ref cRef="betmas:LIT2000Mazmur.11.4">Psalm 11:4</ref>`, but if it works inside the code? Maybe Zhenia will know this.

In itself, the markup inside of `title `of an `msItem` is technically possible (e.g. `Luke <ref cRef="betmas:LIT1812GospelLuke.22.47"> 22,47</ref>`) - but in this case it would create a link that leads nowhere as for the moment there is no text in `LIT1812GospelLuke` with a div for each chapter and verse. Psalms (LIT2000Mazmur) have the `text `, which as a `div `with `n="11"` and` l="4"` so the link to `LIT2000Mazmur.11.4` makes sense